onnxruntime/test/providers/cpu/nn/pool_op_test.cc

```diff
@@ -264,6 +264,51 @@ TEST(PoolTest, MaxPool1D_12_With_Index_8bits) {
   MaxPool1D_12_WithIndexTest_uint8(1 /*storage_order*/);
 }
 
+//template<typename InputIter>
+//void print_vector(std::ostream& os, const std::string& txt, InputIter begin, InputIter end) {
+//  os << txt;
+//  std::copy(begin, end, std::ostream_iterator<std::iterator_traits<InputIter>::value_type>(os, ", "));
+//  os << std::endl;
+//}
+
+// The test on ONNX site is not valid. Either input or output shape is not valid.
+// Given the other things, the output shape must be {1, 1, 8, 5}
```

Comment:
If it is wrong , please speak up now. Let's get it fixed before ONNX 1.7 is released.

Reply:
I will file the issue with ONNX

Reply:
Is this covered by https://github.com/onnx/onnx/issues/2398?

Reply:
It is a different issue

---
In reply to: [400572535](https://github.com/microsoft/onnxruntime/pull/3341#discussion_r400572535) [](ancestors = 400572535)